faq.md

```diff
@@ -7,13 +7,31 @@ css_id: faq
 
 ### **What makes Uptane different from other SOTA security mechanisms?**
 
-
-
+Security problems occur due to accidental disclosures, malicious attacks, disgruntled insiders.  It is not
+a matter of whether a successful attack will occur, but when.  One key feature of a security system is the
+ability to securely recover from an attack.  This means that an update system must have a way to securely 
+recover from a key loss or compromise.  
+
+For example, suppose a nation-state actor steals a signing key and wants to use it to distribute software (as has
+happened before) [cite Iran, etc.].  The update system must provide a way to revoke the current trusted
```

Comment:
Is there a citation link missing?

Reply:
I don't know how to fix the weird breaks. Sorry.